ui/src/components/table/admin-table/table-element/admin-table-columns.tsx

```diff
@@ -0,0 +1,19 @@
+const AdminTableColumns = [
+  {
+    header: 'ADMIN NAME',
+    accessorKey: 'name'
+  },
+  {
+    header: 'UH NUMBER',
+    accessorKey: 'uhUuid'
+  },
+  {
+    header: 'UH USERNAME',
+    accessorKey: 'uid'
+  },
+  {
+    header: 'REMOVE',
+  }
+];
```

Comment:
Let's uncapitalize all the headers since the shadcn/ui Table.tsx component was updated to have a tailwind css class to set the headers to all caps. So the headers would be Admin Name, UH Number, UH Username, and Remove.